subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -0,0 +1,222 @@
+#include <aws_jobs.h>
+#include <aws_fota.h>
+#include <json.h>
+#include "aws_fota_internal.h"
+
+
+static struct pollfd fds;
+
+/* Enum for tracking the job exectuion state */
+static enum execution_status execution_state;
+
+/* Allocated strings for checking topics */
+static u8_t notify_next_topic[NOTIFY_NEXT_TOPIC_LEN + 1];
+static u8_t job_id_update_topic[JOB_ID_UPDATE_TOPIC_MAX_LEN + 1];
+static char version[CONFIG_VERSION_STRING_MAX_LEN + 1];
```

Comment:
Where are these `CONFIG_*` defined?

Reply:
It should be in Kconfig of the library but forgot to include the change :)

Reply:
This should now have been addressed.